spk/prowlarr/src/service-setup.sh

```diff
@@ -0,0 +1,61 @@
+
+# Prowlarr service setup
+
+PROWLARR="${SYNOPKG_PKGDEST}/share/Prowlarr/bin/Prowlarr"
+
+# Prowlarr uses custom Config and PID directories
+HOME_DIR="${SYNOPKG_PKGVAR}"
+CONFIG_DIR="${SYNOPKG_PKGVAR}/.config"
+PID_FILE="${CONFIG_DIR}/Prowlarr/prowlarr.pid"
+
+# Some have it stored in the root of package
+LEGACY_CONFIG_DIR="${SYNOPKG_PKGDEST}/.config"
```

Comment:
There should not be a legacy config directory to worry about since this is the first package.